ffmpeg-2404-sdk/snapcraft.yaml

```diff
@@ -473,3 +473,12 @@ parts:
       - -usr/lib/*/libxcb-x*.so*
       - -usr/lib/*/libxcb-p*.so*
       - -usr/lib/*/libxkb*
+  cleanup:
+    plugin: nil
+    source: https://github.com/canonical/gpu-snap.git
+    source-depth: 1
+    after: [debs]
+    override-prime: |
+      craftctl default
+      sed -i '/usr\/lib\/x86_64-linux-gnu\/libvulkan.so.1\*/d' ${CRAFT_PART_SRC}/lists/*.list
```

Comment:
```suggestion
      sed -i '|usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/libvulkan.so.1\*|d' ${CRAFT_PART_SRC}/lists/${CRAFT_ARCH_BUILD_FOR}.list
```

Reply:
Sorry I needed to close this, as I can't force push to branches under Snapcrafters. I'll reopen this from my fork.